### ISSUE TITLE
Add hyperlink to event description

### ISSUE THREAD

**Issue by cseas** [mozilla/community-portal#506]
I've added the link for the session recording of the event at [this event page](https://community.mozilla.org/events/mozilla-bangalore-hunt-from-home-phase-2/) but it's not rendering as normal text instead of a hyperlink.

I tried writing in Markdown but the `<>` and `[]()` syntaxes don't work either.

Is there a way to add a hyperlink to the description?

I'm not able to add the link to the event details under `Location Details` either due to existing bug #499

**Comment by davesenior** [comment]
We're currently refactoring the event code. Links in description text are currently not included as a feature, but we will mark it as an enhancement. Keep an eye on the tracker for the previous bug.

**Comment by Mte90** [comment]
I don't think that is good to have links as anyone can write there and could be spambot.
